src/components/app/Home.js

```diff
@@ -80,6 +80,17 @@ class UploadButton extends PureComponent {
   }
 }
 
+function InstructionTransition(props: {}) {
+  return (
+    <CSSTransition
+      {...props}
```

Comment:
From the documentation, it's necessary to pass all props along as the `TransitionGroup` tag communicate with props with the underlying `Transition` tag.